drivers/pwm/pwm_mchp_xec_bbled.c

```diff
@@ -28,16 +28,15 @@ LOG_MODULE_REGISTER(pwmbbled_mchp_xec, CONFIG_PWM_LOG_LEVEL);
 #define XEC_PWM_BBLED_INPUT_FREQ_HI	48000000
 #define XEC_PWM_BBLED_INPUT_FREQ_LO	32768
 
-#define XEC_PWM_BBLED_MAX_FREQ_DIV	256U
```

Comment:
Keep this and used it on new macros XEC_PWM_BBLED_MAX_PWM_FREQ_HI

Reply:
Ok.